CMakeLists.txt

```diff
@@ -84,6 +84,13 @@ set(CPACK_COMPONENT_EXAMPLES_DESCRIPTION
     "eProsima Fast DDS doxygen documetation")
 set(CPACK_COMPONENTS_ALL ${CPACK_COMPONENTS_ALL} doxygen)
 
+####################################################################################################
+# Find Sphinx
+####################################################################################################
+set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/code/cmake" ${CMAKE_MODULE_PATH})
+find_package(Sphinx REQUIRED)
+
+set(SPHINX_SOURCE "${CMAKE_SOURCE_DIR}/docs")
```

Comment:
Looking at this more closely, I think this belongs in the `Build Sphinx documentation` section.